src/lib/constants/constants.ts

```diff
@@ -706,3 +706,263 @@ export const aboutTableData = [
 		ipfs: 'Cryptographic validation of all content'
 	}
 ];
+
+export const contributorsList = [
```

Comment:
Hi @grmbyrn I would rename `contributorsList` to `defaultContributorsList` or `staticContributorsList` or `initalContributorsList` to hint it would be fetched dinamically later.